src/components/EventResourceCombobox.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <p-combobox
+    v-model="selected"
+    :options
+    :multiple
+    allow-unknown-value
+    empty-message="All resources"
+    class="event-resource-combobox"
+  />
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+
+  const selected = defineModel<string | string[] | null | undefined>('selected', { required: true })
+
+  const props = defineProps<{
+    multiple?: boolean,
+    additionalOptions?: RelatedResourceGroup[],
```

Comment:
oss doesn't have webhooks. So allowing for additional options to be passed in at implementation. This is so that cloud can define its own version of this component and provide it.